Parser/tokenizer.c

```diff
@@ -1399,7 +1405,9 @@ tok_get(struct tok_state *tok, char **p_start, char **p_end)
     /* Newline */
     if (c == '\n') {
         tok->atbol = 1;
-        if (blankline || tok->level > 0) {
+        if ((blankline || tok->level > 0) &&
+            !(tok->prompt != NULL && tok->nextprompt == NULL &&
+              tok->lineno == 1)) {
```

Comment:
PEP 7 says that in this case (because the condition is broken into multiple lines) the curly brace needs to go on the next line.